### ISSUE TITLE
Optimization of last convolutional layer

### ISSUE THREAD

**Issue by amejs** [jakeret/tf_unet#62]
Hello!

I've just noticed that "weight" and "bias" from the last convolutional layer are not included in "variables", aren't they supposed to be optimized as well? Is there any reason for not including them?

Best regards, Amelia.

**Comment by jakeret** [comment]
You're right, this incorrect. Luckily, this has no effect on the network.
The 'variables' are only used to compute the normalized gradients in order
to visuize the overall training process.
If you have time a would greatly appreciate a PR that fixes the issue ;-)
